### PR TITLE
feature: add anki-apkg-export

### PR DESCRIPTION
**Pull request by yakhinvadim**

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import * as R from 'ramda'
 import debounce from 'lodash.debounce'
+import AnkiExport from 'anki-apkg-export'
```
Comment (ewnd9) [inline src/components/App/App.js:4]: Hi, stumbled upon this pull request in activity view, you probably want to import it dynamically before generating an archive, `sql.js` weights quite a lot, see https://github.com/ewnd9/anki-apkg-export-app/blob/5074907fc9c14d6d9bc5655b610e710cad6a0a41/src/components/main/main.js#L45-L48

Comment (yakhinvadim) [reply src/components/App/App.js:4]: Hi! Oh boy, +1.2mb to bundle size! I'll import it dynamically, thanks for the tip :)

Comment (ewnd9) [reply src/components/App/App.js:4]: Also for your consideration:

Anki as a microservice

https://github.com/fpapado/ephemeral/blob/013906d294e930e041a50ab7f7d34e0f22bb14df/src/export/export.ts#L40-L52

https://github.com/fpapado/micro-anki

Comment (yakhinvadim) [reply src/components/App/App.js:4]: Wow, that's great. Thank you.

**Pull request merged**